src/surrogate_heat_driver.cpp

```diff
@@ -11,8 +11,10 @@
 
 namespace enrico {
 
-SurrogateHeatDriver::SurrogateHeatDriver(MPI_Comm comm, pugi::xml_node node)
-  : Driver(comm)
+SurrogateHeatDriver::SurrogateHeatDriver(MPI_Comm comm,
+                                         double pressure,
+                                         pugi::xml_node node)
+  : HeatFluidsDriver(comm, pressure)
```

Comment:
Since `pressure_` is stored on `HeatFluidsDriver`, the check that `pressure > 0` should probably be contained there. Right now, it's only in `OpenmcNekDriver`.

Reply:
Additionally,  should we remove the `pressure_` member from `OpenmcNekDriver`?  It is only used to instantiate a `NekDriver`.

Reply:
Yes, good catch on that